cmds/coredhcp/main.go

```diff
@@ -20,6 +20,7 @@ import (
 	pl_dns "github.com/coredhcp/coredhcp/plugins/dns"
 	pl_file "github.com/coredhcp/coredhcp/plugins/file"
 	pl_leasetime "github.com/coredhcp/coredhcp/plugins/leasetime"
+	pl_mtu "github.com/coredhcp/coredhcp/plugins/mtu"
```

Comment:
These need to be kept in sync between `cmds/coredhcp-generator` and `cmds/coredhcp` (this is the cause of the build failures)

Reply:
ah, already fixed, thanks!

Reply:
Hi! yes, I missed that one 😉